### DISCUSSION TITLE
Do the Akka/Pekko courses only use `Akka Typed` and `Pekko Typed`?

### DISCUSSION THREAD

**Discussion by andreilgeorgescu** [riverwalk-software/astro-test#79] [Q&A] [answered]
@daniel-ciocirlan if so then I think it makes sense to title them all variations of `Akka/Pekko Typed`.

**Comment by daniel-ciocirlan** [comment, marked as answer]
No need, the "typed" label is relevant to the core module (actors) only, and I'm willing to cut it from that course name as well (the typed actor API has been available & preferred for 5 years).
